docs/docs/docs/01-core/admin/06-commands/02-develop.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Develop
+tags:
+  - commands
+  - admin
+  - develop
+---
+
+## How it works
+
+The develop command set's itself up much like the [build](build) command. Once we've injected our middlewares we load the strapi instance and then generate the types based off the user's instance as well as potentially compiling any TS server code if we're in a TS project. The final step is to watch the project directory so we can restart the strapi instance in real-time as a user is developing their project & being the actual strapi instance.
+
+## CLI Usage
+
+```bash
+strapi develop
+```
+
+### Options
+
+```bash
+Start your Strapi application in development mode
+
+Options:
+  --polling         Watch for file changes in network directories (default: false)
```

Comment:
Network directories probably needs further clarification.

Reply:
Yeah I didn't really understand this, this is from the current watch command and is passed to chokidar, I'll look into it.

Reply:
I didn't understand it either honestly lol, but if it applies to both server and admin polling I think we should enable this by default